pkg/kubepkg/kubepkg.go

```diff
@@ -343,54 +317,60 @@ func (c cfg) run() error {
 		}
 	}
 
-	err = runCommand(dstdir, "dpkg-buildpackage", "-us", "-uc", "-b", "-a"+c.DebArch)
-	if err != nil {
-		return err
+	logrus.Infof("Running dpkg-buildpackage for %s (%s/%s)", c.Package, c.Arch, c.DebArch)
+
+	dpkgStatus, dpkgErr := command.NewWithWorkDir(dstdir, "dpkg-buildpackage", "-us", "-uc", "-b", "-a"+c.DebArch).Run()
```

Comment:
Just to make sure this works, we do not need an `=` for the `-a` parameter here, right?

Like: `"-a="+c.DebArch`